docs/vsphere.md

```diff
@@ -11,7 +11,7 @@ Kubespray can be deployed with vSphere as Cloud provider. This feature supports:
 
 ### Prerequisites
 
-You need at first to configure your vSphere environment by following the [official documentation](https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md#prerequisites).
+You need at first to configure your vSphere environment by following the [documentation](https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md#prerequisites).
```

Comment:
Not sure this PR is really needed, only changing "official documentation" to "documentation" seems a bit overkill ?

Reply:
It's my bad, when i reacreted commit I just got confused, resending.